api/utils.py

```diff
@@ -867,3 +867,28 @@ def check_fairsharing_abbreviation(fairlist, abreviation):
         if abreviation == standard["attributes"]["abbreviation"]:
             return (100, "Your metadata standard appears in Fairsharing")
     return (0, "Your metadata standard has not been found in Fairsharing")
+
+
+def check_ORCID(orcid):
+    headers = {
+        "Accept": "application/xml",
+    }
+
+    response = requests.get(orcid, headers=headers)
```

Comment:
ORCID is not a URL, what is this method expecting as input argument?

Besides the FAIR EVA code already imports [`idutils` library which has a `is_orcid()` method](https://idutils.readthedocs.io/en/latest/); it is better to use this one

Reply:
ORCID is sometimes expressed as a URL. Now it can support both cases

Reply:
Sadly for the time being `idutils.is_orcid()` does not work with some  URLS like https://orcid.org/0000-0001-5750-0872/Project_leader but it does work with https://orcid.org/0009-0000-2496-4136 so i am not sure on what to do

Reply:
Solved, check_orcid() is removed, cases not recognised by is_orcid will not ve accepted